client/src/stories/leaderboard.stories.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import Leaderboard from '../components/Leaderboard/Leaderboard';
+
+
+export default { title: 'Leaderboard' }
+
+export const leaderboard = () => <Leaderboard users={leaderboardData} />
+
+
+
+const leaderboardData = [
```

Comment:
⬆️ All that whitespace above that line tells me you're not using `Prettier` in your editor :)

Reply:
its weird, it used to auto format on save, but it stopped working. Need to check it out again